server/apps/publish/formatters/aap_ipnews_formatter.py

```diff
@@ -83,6 +83,7 @@ def format(self, article, subscriber):
             odbc_item['priority'] = article.get('priority', 'r')  # @priority
             odbc_item['service_level'] = 'a'  # @service_level
 
+            odbc_item['selector_codes'] = '3**'
```

Comment:
Need to have something or the IPNews client fails, this will change once we get the mapping implemented somewhere!